react/__tests__/FilterNavigator.test.js

```diff
@@ -1,34 +1,40 @@
+/* eslint-disable jest/no-mocks-import */
 /* eslint-env jest */
 import React from 'react'
-import { render } from '@vtex/test-tools/react'
-
+import { render, fireEvent } from '@vtex/test-tools/react'
 import categoriesTree from 'categoriesTree'
+import { useDevice } from 'vtex.device-detector'
 
 import FilterNavigator from '../FilterNavigator'
 import QueryContext from '../components/QueryContext'
-
 import { useRuntime } from '../__mocks__/vtex.render-runtime'
 import FilterNavigatorContext from '../components/FilterNavigatorContext'
+import specifications from '../__mocks__/specifications'
+
 const mockUseRuntime = useRuntime
 
 const mockNavigate = jest.fn()
 const mockSetQuery = jest.fn()
+
 beforeEach(() => {
   jest.clearAllMocks()
 
   mockUseRuntime.mockImplementation(() => ({
     navigate: mockNavigate,
     setQuery: mockSetQuery,
     getSettings: () => ({}),
+    query: {},
   }))
+
+  useDevice.mockImplementation(() => ({ isMobile: false }))
 })
 
 describe('<FilterNavigator />', () => {
   const renderComponent = (customProps = { query: 'clothing' }) => {
```

Comment:
```suggestion
  const renderComponent = (customProps = { query: 'clothing', map: 'c' }) => {
```
so you don't have to check the map below :)